lib/_stream_writable.js

```diff
@@ -113,7 +106,9 @@ function WritableState(options, stream) {
 
   // allocate the first CorkedRequest, there is always
   // one allocated and free to use, and we maintain at most two
-  this.corkedRequestsFree = new CorkedRequest(this);
+  var corkReq = { next: null, entry: null, finish: undefined };
+  corkReq.finish = onCorkedFinish.bind(undefined, corkReq, this);
+  this.corkedRequestsFree = corkReq;
```

Comment:
Is this giving any perf increase? The major benefit is to avoid the creation of `CorkedRequest`, which was introduced to leverage hidden classes and the like.